src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala

```diff
@@ -921,4 +940,13 @@ class StammdatenUpdateService(override val sysConfig: SystemConfig) extends Even
       }
     }
   }
+
+  private def getUpdatedDescriptionLieferplanung(abotypDepotTour: String, newName: String, oldName: String): String = {
+    val abotypDepotTourReplaced = abotypDepotTour
+      .replaceAll(oldName + ',', newName)
```

Comment:
here the `oldname,` is replaces by `newName` (no comma). Same line below.

Reply:
you're totally right. Changed it